tests/i18n/tests.py

```diff
@@ -259,8 +259,21 @@ def test_maclines(self):
             self.assertEqual('Catalan Win\nEOF\n', gettext('Win\r\nEOF\r\n'))
 
     def test_to_locale(self):
-        self.assertEqual(to_locale('en-us'), 'en_US')
-        self.assertEqual(to_locale('sr-lat'), 'sr_Lat')
+        """
+        Tests the to_locale function and the special case of Serbian Latin as well as subtags
```

Comment:
The inline comments are sufficient. I don't think the docstring is adding additional value.